CONTRIBUTING.rst

```diff
@@ -0,0 +1,44 @@
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS `Developer Documentation`_ to get started.
+
+.. _Developer Documentation: https://birdhouse-deploy.readthedocs.io/en/latest/
+
+Policy
+------
+Since PAVICS is used in production by multiple organizations, this deployment repository also has a policy regarding contributions.
+
+Policy objectives
+~~~~~~~~~~~~~~~~~~~~~
+
+1. Keep production stable between deployments
+2. Encourage PR to be merged more quickly
+3. Achieve the two previous objectives
+      * by weighing down everyone's workflow as little as possible
+      * by having an approach that scales well as more nodes go into production and the number of PR increases
+
+Policy rules
+~~~~~~~~~~~~~~~~~~~~~
+
+1. The repository has a main branch, "master", open to the community where contributions, "PR", are welcome. This master branch must not have owners and therefore no organization can block contributions to it.
+2. Contributions should be backward-compatible whenever possible, or feature a toggle switch so that organizations can activate the new feature on their own schedule.
+3. Contributions will trigger a test suite that must successfully pass before being merged (or integrated).
+4. Contributions must be reviewed by at least one organization, and possibly more than one organization depending on the impact of the contribution.
```

Comment:
This will need to be investigated more. Maybe define a list of default reviewers?
I'm trying to consider the point of view of a new contributor. They will have no idea which organization, and even less which key person from them, to add when submitting a new PR / feature. 
How can they know who is impacted or depends on a given change?

Reply:
> How can they know who is impacted or depends on a given change?

In fact, given rule 6, maybe we could state that contributions must be reviewed by every willing organizations and then state a default reviewer per org. I propose @tlvu for Ouranos and @MatProv for CRIM.